Pod/Classes/BONChain.m

```diff
@@ -210,6 +210,28 @@ - (BONChainIndentSpacer)indentSpacer
     return [indentSpacerBlock copy];
 }
 
+- (BONChainUnderlineStyle)underlineStyle
+{
+  BONChainUnderlineStyle underlineStyleBlock = ^(NSUnderlineStyle style) {
```

Comment:
#req please use four spaces to indent. See the [Contributing](https://github.com/Raizlabs/BonMot#contributing) section of the ReadMe if you want to use `clang-format` to double check the formatting.